third_party/gpus/cuda_configure.bzl

```diff
@@ -668,6 +669,10 @@ def _get_cuda_config(repository_ctx, find_cuda_config_script):
     cudnn_version = ("64_%s" if is_windows else "%s") % config["cudnn_version"]
 
     if int(cuda_major) >= 11:
+        if int(cuda_major) == 11:
+          cudart_version = "64_110" if is_windows else "11.0"
```

Comment:
Can you please add a comment here on why we need this special logic (same text as the PR description is fine).

Reply:
Done.